### ISSUE TITLE
Number Features in Compound Map Tools needs test written

### ISSUE THREAD

**Issue by mfunk** [Esri/solutions-geoprocessing-toolbox#539]
The **Number Features** tool in *Compound Map Tools_xxxx* needs tests written. This is the SAME tool that is in the *Clearing Operations Tools_xxxx* toolbox. If there is a way to reuse or point to that same test it would make it easier to test all 4 toolboxes.

## Expected Behavior
We want to make sure that all of the tools in the toolboxes have some kind of automated test making sure they aren't failing.

## Current Behavior
New toolbox to repo, no test exists yet.

## Possible Solution
Reuse test from *Clearing Operations Tools_xxxx*.

## Context
Basically the goal of all automated testing is to find defects in tools repeatedly in an automated fashion so they can be easily tested across operating systems, platforms, client apps, and software versions.

**Comment by topowright** [comment]
This has been competed.
